### PR TITLE
Op 372 / Added Sylius 1.12 & 1.13 support

### PR DESCRIPTION
**Pull request by tomkalon**
+ added Sylius 1.12 & 1.13 support
+ Gulp changed to Webpack
+ Upgrade: Node.js 10 -> Node.js 14
+ Flashbag -> RequestStack
+ ECS fixes -> CodingStandard 3.0
+ Readme.md has been updated

### CODE REVIEW

File: features/exporting_shipping_data_to_api_poczta_polska.feature [hunk 1]
```diff
@@ -34,14 +34,14 @@ Feature: Managing shipping gateway
...
-        And the store has a product "Chicken" priced at "$2" in "Web-US" channel
+        And the store has a product "Chicken" priced at "$2.00" in "Web-US" channel
         And customer "user@bitbag.pl" has placed 5 orders on the "Web-US" channel in each buying 5 "Chicken" products
         And the customer set the shipping address "Mike Ross" addressed it to "350 5th Ave", "10118" "New York" in the "United States" to orders
         And those orders were placed with "PocztaPolska Express" shipping method
         And set product weight to "10"
         And set units to the shipment
 
-    @ui
+    @ui @test
```
Comment (senghe) [inline features/exporting_shipping_data_to_api_poczta_polska.feature]: What's the @test?

File: src/EventListener/ShippingExportEventListener.php [hunk 2]
```diff
@@ -82,26 +83,26 @@ public function exportShipment(ResourceControllerEvent $event): void
 
             $this->saveShippingLabel($shippingExport, $labelContent, 'pdf');
         } catch (SoapFault $exception) {
-            $this->flashBag->add(
+            $this->requestStack->getSession()->getBag('flashes')->add(
```
Comment (senghe) [inline src/EventListener/ShippingExportEventListener.php]: I would assign the flashbag to a variable

File: tests/Application/.babelrc
Comment (senghe) [inline tests/Application/.babelrc:1]: Why this file has been removed?

File: tests/Application/config/packages/webpackl_encore.yml
Comment (senghe) [inline tests/Application/config/packages/webpackl_encore.yml:1]: A typo in file name

**Review by senghe — CHANGES_REQUESTED** [review]

File: features/exporting_shipping_data_to_api_poczta_polska.feature (hunk 1 above)
Comment (tomkalon) [reply features/exporting_shipping_data_to_api_poczta_polska.feature]: Fixed. After upgrading to Sylius 1.13 this test was throwing an error. I checked @test to not check the other tests and forgot to delete.

File: src/EventListener/ShippingExportEventListener.php (hunk 2 above)
Comment (tomkalon) [reply src/EventListener/ShippingExportEventListener.php]: Fixed

File: tests/Application/.babelrc
Comment (tomkalon) [reply tests/Application/.babelrc:1]: https://docs.sylius.com/en/1.12/cookbook/frontend/migrating-to-webpack-1-12-or-later.html

> 2. Remove the following files from you project:
> 
> .babelrc
> gulpfile.babel.js
> yarn.lock
> public/assets
> node_modules

File: tests/Application/config/packages/webpackl_encore.yml
Comment (tomkalon) [reply tests/Application/config/packages/webpackl_encore.yml:1]: Fixed

**Review by senghe — APPROVED** [review]

**Pull request merged**